src/librustc/front/map/definitions.rs

```diff
@@ -212,10 +213,6 @@ impl DefPathData {
                 InternedString::new("?")
             }
 
-            Impl => {
-                InternedString::new("<impl>")
-            }
-
```

Comment:
I was hoping this could be combined with the actual type info to produce the full name.
Right now, the `ast::Item` name for impls is a hacky composition of the pretty-printed AST type and trait and I wish we could get rid of it.